src/include/targets.h

```diff
@@ -606,6 +606,22 @@ Designed by NamimnoRC
 #define GPIO_PIN_LED            16
 #define GPIO_PIN_BUTTON         0
 
+#elif defined(TARGET_RX_MATEK_2400)
+#define GPIO_PIN_NSS            15
+#define GPIO_PIN_BUSY           5
+#define GPIO_PIN_DIO1           4
+#define GPIO_PIN_MOSI           13
+#define GPIO_PIN_MISO           12
+#define GPIO_PIN_SCK            14
+#define GPIO_PIN_RST            2
+#define GPIO_PIN_LED            16
+#define GPIO_PIN_BUTTON         0
+#define timerOffset             -1
```

Comment:
This all looks good to me except this timerOffset is no longer used so it can be removed and this has my approval.